OpenRA.Game/Graphics/TerrainSpriteLayer.cs

```diff
@@ -0,0 +1,135 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Graphics
+{
+	public sealed class TerrainSpriteLayer : IDisposable
+	{
+		static readonly Sprite EmptySprite = new Sprite(new Sheet(new Size(1, 1)), Rectangle.Empty, TextureChannel.Alpha);
```

Comment:
This is a bit bogus. It's in its own weird sheet, so if you end up using it, you violate the one-sheet-only invariant. (benignly)